bindings/fluidmax/fluidmax.c

```diff
@@ -1065,7 +1065,7 @@ fluidmax_version(t_object *o)
   post("  Max/MSP integration by Norbert Schnell IMTR IRCAM - Centre Pompidou");
 }
 
-extern fluid_gen_info_t fluid_gen_info[];
+extern const fluid_gen_info_t fluid_gen_info[];
```

Comment:
I see.
If I could say my opinion, perhaps it would be worth to this change to 'static' when you will decide to officially discard this fluidmax thing (I do not know what it is exactly, actually) by doing both actions together... what do you think?

Reply:
I dont think this would mean to discard it. It would just become a bit more "unmaintained". But this decision seems to have been made years ago.

Reply:
Ok, so let me staticallyzise it.